nebula/ui/components/VPNSubscriptionOption.qml

```diff
@@ -21,7 +21,7 @@ RadioDelegate {
     ButtonGroup.group: subscriptionOptions
 
     Layout.fillWidth: true
-    Layout.preferredHeight: 96
+    Layout.preferredHeight: chipFreeTrial.visible ? 96+headLineLabel.overflow : 96
```

Comment:
I think you can achieve the same thing by using the height of the layout, which may be a little more robust. Basically making it at least 96 points tall, but expanding if necessary

something like: `Layout.preferredHeight: Math.max(96, row.implicitHeight + row.anchors.topMargin + row.anchors.bottomMargin)`